app/assets/javascripts/blacklight/autofocus.js

```diff
@@ -1,7 +1,7 @@
 //= require blacklight/core
 (function($) {
-  Blacklight.do_search_autofocus_fallback = function() {
-    if (typeof Modernizer != "undefined") {
+  Blacklight.doSearchAutofocusFallback = function() {
```

Comment:
Should we try to notify users with console warnings for 6.x deprecation messages for the method renames?

Reply:
I guess we could do something like:

```
Blacklight.doSearchAutofocusFallback = function() {
if(typeof Blacklight.do_search_autofocus_fallback == "function") {
  console.log("DEPRECATED")
  return Blacklight.do_search_autofocus_fallback();
}
...
}
```

Right?

Reply:
I was thinking something like:

```javascript
Blacklight.do_search_autofocus_fallback = function() {
  console.warn("DEPRECATED: Blacklight.do_search_autofocus_fallback is deprecated and will be removed in version 7.0 This method has been renamed to Blacklight.doSearchAutofocusFallback")
  Blacklight.doSearchAutofocusFallback();
})
```

Reply:
I thought about that, but `do_search_autofocus_fallback` is a method that people may have overridden. I suppose we might need both, in case someone is calling those methods directly?

Reply:
Err.. yeah I guess that could be problematic.

Reply:
So, we can do that in a backport?